secureboot_policy.go

```diff
@@ -0,0 +1,1292 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package secboot
+
```

Comment:
this file feels a bit too  large, is there a way to split into lower level/higher level bits?  or spit out the low level efi bits and the signature db bits ?

Reply:
I think what I'd like to do in a future PR is split some of this code out in to an internal package - that will make this file quite a bit smaller but also reduce the number of test-only exports that are defined in export_test.go.